Mcode/+logger/Log4jConfigurator.m

```diff
@@ -114,9 +114,9 @@ function setLevels(levels)
             %     });
             for i = 1:size(levels, 1)
                 [logName,levelName] = levels{i,:};
-                logger = org.apache.log4j.LogManager.getLogger(logName);
+                loggerobj = org.apache.log4j.LogManager.getLogger(logName);
```

Comment:
Can we do `loggerObj` instead of `loggerobj`? This project uses camelCase for local variable names.

Reply:
sure, no objection of course.   I am new to the process of contributing -- I think my next steps are to make the change in my repo, then update the PR.
If I succeed in that, then I'll try to do likewise with your other feedbacks.  If I mishandle it though, feel free to reject my PR and do the right thing.

Reply:
notes to self: by doing two steps (a) commit to my local repo, then (b) push to my origin on github -- if I'm using the right words for all that -- the change is now visible in the still-open PR.  No extra effort to get it into the PR.

Reply:
Yep! Make the changes in your repo on your branch. Use `git rebase -i` to "squash" the commits so there's a tidier commit history instead of several fixup/revision commits at the end. And the `git push --force` to push those new changes to your branch in your repo, and they'll automatically show up in this PR for me to see; nothing else you need to do after that except come back here and make more comments. That's the normal way of iterating on a PR that needs changes.